packages/experimental/Drawer/android/.gitignore

```diff
@@ -0,0 +1,5 @@
+# Ignore Gradle project-specific cache directory
+.gradle
+
+# Ignore Gradle build output directory
+build
```

Comment:
Could we push this up to the main repos gitignore via `**/build` or something like that, so that future android packages don't need to remember to add this gitignore?